site/public/js/notebook_builder/builders/abstract-builder.js

```diff
@@ -11,7 +11,7 @@ class AbstractBuilder {
         this.reorderable_widgets_div = document.createElement('div');
         this.itempool_div = document.createElement('div');
 
-        this.selector_options = ['Multiple Choice', 'Markdown', 'Short Answer', 'Image'];
+        this.selector_options = ['Markdown', 'Multiple Choice', 'Short Answer', 'Image'];
```

Comment:
is any specific reason for swapping the `markdown` and `multiple choice`?

Reply:
Yeah, during the discussion with Barb, we came to a conclusion that it would be more intuitive to have it first.﻿